contracts/core/Allo.sol

```diff
@@ -255,6 +255,22 @@ contract Allo is
         _grantRole(pools[_poolId].managerRole, _manager);
     }
 
+    /// @notice Add multiple pool managers
+    /// @dev Emits 'RoleGranted()' event. 'msg.sender' must be a pool admin.
+    /// @param _poolId ID of the pool
+    /// @param _managers The addresses to add
+    function addPoolManagers(uint256 _poolId, address[] memory _managers) external onlyPoolAdmin(_poolId) {
```

Comment:
I think `calldata` is more appropriate than `memory` here. `_managers` won't get modified and using `calldata` is slightly more efficient.